integration_test/third_party_apps_data/applications/aerospike/metadata.yaml

```diff
@@ -0,0 +1,99 @@
+short_name: Aerospike
+long_name: Aerospike
+logo_path: ""
+description: |-
+  The Aerospike integration collects key namespace and system metrics, such as disk and memory usage, scans,
+  and connections. The integration collects these metrics using the official
+  client API provided by Aerospike.
+supported_app_version: ["4.9", "5.x", "6.x"]
+configure_integration: ""
+expected_metrics:
+  - kind: CUMULATIVE
+    labels:
+      result: error|filtered_out|not_found|success|timeout
+      type: delete|read|udf|write
+      node_name: .*
+      namespace_name: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/aerospike.namespace.transaction.count
+    value_type: INT64
+  - kind: GAUGE
+    labels:
+      node_name: .*
+      namespace_name: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/aerospike.namespace.disk.available
+    optional: true
+    value_type: INT64
+  - kind: GAUGE
+    labels:
+      node_name: .*
+      namespace_name: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/aerospike.namespace.memory.free
+    value_type: INT64
+  - kind: GAUGE
+    labels:
+      component: data|index|set_index|secondary_index
+      node_name: .*
+      namespace_name: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/aerospike.namespace.memory.usage
+    value_type: INT64
+  - kind: CUMULATIVE
+    labels:
+      result: abort|complete|error
+      type: aggregation|basic|ops_background|udf_background
+      node_name: .*
+      namespace_name: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/aerospike.namespace.scan.count
+    value_type: INT64
+  - kind: CUMULATIVE
+    labels:
+      operation: close|open
+      type: client|fabric|heartbeat
+      node_name: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/aerospike.node.connection.count
+    value_type: INT64
+  - kind: GAUGE
+    labels:
+      type: client|fabric|heartbeat
+      node_name: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/aerospike.node.connection.open
+    value_type: INT64
+    representative: true
+  - kind: GAUGE
+    labels:
+      node_name: .*
+    monitored_resource: gce_instance
+    type: workload.googleapis.com/aerospike.node.memory.free
+    value_type: INT64
+configuration_options:
+  metrics:
+  - type: aerospike
+    fields:
+    - name: type
+      default: null
+      description: This value must be aerospike.
+    - name: collection_interval
+      default: 60s
+      description: A time.Duration value, such as 30s or 5m.
+    - name: endpoint
+      default: localhost:3000
+      description: The JMX Service URL or host and port used to construct the service URL. This value must be in the form of service:jmx:<protocol>:<sap> or host:port. Values in host:port form are used to create a service URL of service:jmx:rmi:///jndi/rmi://<host>:<port>/jmxrmi.
```

Comment:
The [JMX Service URL](https://docs.oracle.com/javase/8/docs/api/javax/management/remote/JMXServiceURL.html) or host and port used to construct the service URL. This value must be in the form of `host:port`, which is used to create a service URL of the form `service:jmx:remote+http://<host>:<port>/jmxrmi`.